code/game/objects/items/stacks/medical.dm

```diff
@@ -417,3 +417,42 @@
 	custom_materials = null
 	is_cyborg = 1
 	cost = 250
+
+/obj/item/stack/medical/slime_patch
+	name = "slime patch"
+	singular_name = "slime patch"
+	desc = "Potent patch containing smart chemicals, able to detect and fill out even inorganic damage."

```

Comment:
might wanna change this description to not mention chemicals and change the name to not use the word "patch", as the term "patch" in ss13 chemistry usually refers to chem patches, and that could cause some confusion

also, if possible, could you allow these to heal golems and skeletons, if they (and/or normal sutures?) can't heal them already?